pkg/kubelet/cm/deviceplugin/pod_devices.go

```diff
@@ -191,57 +191,60 @@ func (pdev podDevices) deviceRunContainerOptions(podUID, contName string) *Devic
 	devsMap := make(map[string]string)
 	mountsMap := make(map[string]string)
 	envsMap := make(map[string]string)
+	annotationsMap := make(map[string]string)
 	// Loops through AllocationResponses of all cached device resources.
 	for _, devices := range resources {
 		resp := devices.allocResp
 		// Each Allocate response has the following artifacts.
 		// Environment variables
 		// Mount points
 		// Device files
+		// Annotations
 		// These artifacts are per resource per container.
 		// Updates RunContainerOptions.Envs.
 		for k, v := range resp.Envs {
 			if e, ok := envsMap[k]; ok {
-				glog.V(3).Infof("skip existing env %s %s", k, v)
+				glog.V(4).Infof("Skip existing env %s %s", k, v)
 				if e != v {
-					glog.Errorf("Environment variable %s has conflicting setting: %s and %s", k, e, v)
+					glog.Errorf("environment variable %s has conflicting setting: %s and %s", k, e, v)
```

Comment:
We should start with upper case for both info and error logs. The lower case is for `err` objects which are returned. https://github.com/golang/go/wiki/CodeReviewComments#error-strings

Reply:
Looks like the kubelet module (not sure about kubernetes as a project) has a different convention:
https://github.com/kubernetes/kubernetes/blob/master/pkg/kubelet/kubelet.go

Reply:
kubelet package adheres to the same convention for logs. Errors (not logs) start with a lower case.

Reply:
All the 17 occurences of `glog.Errorf` in https://github.com/kubernetes/kubernetes/blob/master/pkg/kubelet/kubelet.go start with a upper case letter. And all but 3 occurences of `fmt.Errorf` start with a lower case letter. So, it seems to be mostly following the Go guidelines.

Reply:
Yep sorry looks like I reversed the Infof and Errorf again